aks/README.md

```diff
@@ -44,7 +45,7 @@
       nginx.ingress.kubernetes.io/proxy-body-size: 1024m
       certmanager.k8s.io/cluster-issuer: letsencrypt-prod
       certmanager.k8s.io/acme-challenge-type: dns01
-      certmanager.k8s.io/acme-dns01-provider: azuredns
+      certmanager.k8s.io/ace-dns01-provider: azuredns
```

Comment:
Typo perhaps? This has to be `acme` as in the https://en.wikipedia.org/wiki/Automated_Certificate_Management_Environment protocol.

Reply:
Fixed in 46fb21b .